src/backend/oracle-data-api/pom.xml

```diff
@@ -6,7 +6,7 @@
 	<parent>
 		<groupId>org.springframework.boot</groupId>
 		<artifactId>spring-boot-starter-parent</artifactId>
-		<version>2.6.4</version>
+		<version>2.7.0</version>
```

Comment:
Can we actually pull out the changes to the pom.xml from this PR?  This way we can separate the UUID change from the spring upgrade PR. Once this PR has been merged into develop, we can rebase the dependabot PR and merge it in.

Reply:
Sure, will change this back to 2.6.4 and get the spring upgrade from the dependabot PR after merging this one.